src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -68,11 +68,6 @@ const BrokerFormContent: FC<Props> = ({
     })
   }, [])
 
-  useEffect(() => {
-    if (formContent.brokerUsername) {
-      setSecurity('user')
-    }
```

Comment:
In order to avoid creating a dependency cycle between this PR and the IDPE one, could we temporarily bring this change back and add the following conditional to handle the certificate scenario? That way this could be merged without `authType` existing. Don't lose the commit because I like what you've changed here with the `authType` integration but I think we could make that change in a followup PR.

```
    if (formContent.brokerUsername) {
      setSecurity('user')
    }
    if (formContent.brokerClientCert) {
      setSecurity('certificate')
    }
```

Reply:
nevermind, Subir was able to merge the authType property to nifid. So this no longer applies